.github/workflows/integration-test.yml

```diff
@@ -30,10 +30,13 @@ jobs:
       - name: Install go-junit-report
         run: go install github.com/jstemmer/go-junit-report@latest
 
+      - name: Set env var for ablytest
+        run: echo "ABLY_PROTOCOL=${{ matrix.protocol == 'json' && 'application/json' || 'application/x-msgpack' }}" >> $GITHUB_ENV
```

Comment:
This could just be done within the script below rather than a separate step:

```yaml
run: |
  set -o pipefail
  export ABLY_PROTOCOL=${{ matrix.protocol == 'json' && 'application/json' || 'application/x-msgpack' }}
  go test -tags=integration ...

```